test/runtests.cmd

```diff
@@ -474,6 +475,7 @@ goto :main
 
   echo ^>^> %*
   cmd /s /c "%*"
+  if %ERRORLEVEL% NEQ 0 exit /b %ERRORLEVEL%
 

```

Comment:
lol, oh yeah.  my bad, you're right.

let us... stop using batch scripts as soon as reasonably possible.